rogue-thi-app/lib/backend-utils/rooms-utils.js

```diff
@@ -91,11 +91,15 @@ export function getRoomOpenings (rooms, date) {
           room: room === 0 ? ROOMS_ALL : room,
           type: stunde.type,
           from: new Date(stunde.von),
-          until: new Date(stunde.bis)
+          until: new Date(stunde.bis),
+          capacityData: stunde.raeume
```

Comment:
Additionally, u can just unpack the capacity data while iterating the room data like this:
```js
.flatMap(stunde =>
      stunde.raeume
        .map(([,, room, capacity]) => {
          return ({
            // 0 indicates that every room is free
            room: room === 0 ? ROOMS_ALL : room,
            type: stunde.type,
            from: new Date(stunde.von),
            until: new Date(stunde.bis),
            capacity: Number(capacity) || null
          })
        })
    )
```